source/node-lib-qt-rss/main.cpp

```diff
@@ -34,7 +34,12 @@ int main(int argc, char* argv[]) {
         return -1;
     }
 
-    QObject::connect(&engine, &QQmlEngine::quit, [](){ node::lib::StopEventLoop(); });
+    QObject::connect(&engine,
+            &QQmlEngine::quit,
+            [](){
+                node::lib::StopEventLoop();
+                node::lib::Deinitialize();
```

Comment:
StopEventLoop() shouldn't be required anymore and Deinitialize() could be moved to line 53, after app.exec() returns. This lambda was only required because node had the control, if I remember correctly.